### DISCUSSION TITLE
Possibility for more than one consume folder?

### DISCUSSION THREAD

**Discussion by Starky** [jonaswinkler/paperless-ng#844] [Support] [answered]
(Installed via Docker compose.)

I'd like to let Paperless-ng watch more than one folder for documents (for example my scan and work folders) but it seems that there's only one folder supported. Is this correct? Is there also any possibility to let it check subfolders? Thanks in advance, great work!

**Comment by jonaswinkler** [comment, marked as answer]
Only one consume folder, yes.

However, subfolders are supported. You need to enable that. Search the documentation at https://paperless-ng.readthedocs.io/en/latest/configuration.html for PAPERLESS_CONSUMER_RECURSIVE.
